sklearn/model_selection/_search.py

```diff
@@ -204,15 +204,24 @@ class ParameterSampler(object):
 
     Parameters
     ----------
-    param_distributions : dict
+    param_distributions : dict or sequence of such.
         Dictionary where the keys are parameters and values
         are distributions from which a parameter is to be sampled.
         Distributions either have to provide a ``rvs`` function
         to sample from them, or can be given as a list of values,
         where a uniform distribution is assumed.
 
-    n_iter : integer
+    n_iter : integer or sequence of such
```

Comment:
hmm is not better use `array like`?

Reply:
or list

Reply:
I agree with you, I was just trying to keep it similar  ```param_distributions```
```suggestion
    n_iter : integer or list of integers
```